src/curl.cpp

```diff
@@ -346,7 +346,7 @@ long             S3fsCurl::connect_timeout     = 300;  // default
 time_t           S3fsCurl::readwrite_timeout   = 60;   // default
 int              S3fsCurl::retries             = 3;    // default
 bool             S3fsCurl::is_public_bucket    = false;
-string           S3fsCurl::default_acl         = "private";
+string           S3fsCurl::default_acl         = "";
```

Comment:
I do not think s3fs should send `x-amz-acl` if the user does not set a value.  No value corresponds to private access, from https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPUT.html:

> Additionally, you can use the following access control–related headers with this operation. By default, all objects are private: only the owner has full control.

This will ensure that partial S3 implementations like StorageGRID work without additional configuration.